docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,321 @@
+
+#### Images
+
+Technology Partners can add up to eight images (seven if you are including a video) in a tile's media carousel.
+
+The images must meet the following requirements:
+
+| Image Requirements | Description                                                                                                                                       |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Type               | `.jpg` or `.png`.                                                                                                                                 |
+| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
+| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
+| Name               | Use letters, numbers, underscores, and hyphens. Do not use spaces.                                                                           |
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the `manifest.json` file which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][22].
+
+## Open a pull request
+
+Before you open a pull request, run the following command to catch any problems with your integration:
+
+```
+ddev validate all <INTEGRATION_NAME>
+```
+
+Complete the following steps:
+
+1. Commit all changes to your feature branch.
+2. Push your changes to the remote repository. 
+3. Open a pull request that contains your integration tile's asset files (including images) in the [`marketplace`][18] or [`integrations-extras`][26] repository. 
```

Comment:
If you don't want to mention both repos here, you could use tabs around the "Open a pull request" section